website/content/api-docs/secret/databases/index.mdx

```diff
@@ -166,7 +169,10 @@ $ curl \
       "connection_url": "{{username}}:{{password}}@tcp(127.0.0.1:3306)/",
       "username": "vaultuser"
     },
-    "plugin_name": "mysql-database-plugin"
+    "password_policy": "",
+    "plugin_name": "postgresql-database-plugin",
```

Comment:
Can we change this to `mysql-database-plugin`, or change the curl command above to `http://127.0.0.1:8200/v1/database/config/postgres`? Just so they match.